pkg/operator/controller/controller_default_certificate.go

```diff
@@ -86,7 +81,7 @@ func desiredRouterDefaultCertificateSecret(ca *crypto.CA, deployment *appsv1.Dep
 		return nil, fmt.Errorf("failed to encode certificate: %v", err)
 	}
 
-	name := routerDefaultCertificateSecretName(ci, deployment.Namespace)
+	name := routerDefaultCertificateSecretName(ci, namespace)
```

Comment:
Could use `deploymentRef.Namespace` to obviate the separate `namespace` parameter.

Reply:
Unfortunately OwnerReference does not contain Namespace field.

Reply:
Whoops! Sorry! Hmm, I think that's why I decided to pass `deployment` before, to avoid needing to pass two arguments (`deploymentRef` and `namespace`).